src/view.js

```diff
@@ -215,6 +119,6 @@ const View = Backbone.View.extend({
   setDomApi
 });
 
-_.extend(View.prototype, ViewMixin, RegionsMixin);
+_.extend(View.prototype, ViewMixin, RegionsMixin, TemplateRenderMixin);
```

Comment:
I went ahead a POC'd the TemplateRenderMixin.  I like it a lot.. we'll want to merge quite a few of the other open PRs before actually implementing this, but it looks really nice to me.